test/fixtures/full/main.tf

```diff
@@ -41,19 +41,29 @@ resource "random_string" "suffix" {
   upper   = false
 }
 
-resource "google_compute_network" "network" {
-  name                    = "pf-test-full-${random_string.suffix.result}"
-  routing_mode            = "GLOBAL"
-  auto_create_subnetworks = "false"
-  project                 = "${var.shared_vpc}"
-}
+module "vpc" {
+  source          = "terraform-google-modules/network/google"
+  version         = "~> 0.4.0"
+  network_name    = "pf-test-int-full-${random_string.suffix.result}"
+  project_id      = "${var.shared_vpc}"
+  shared_vpc_host = "false"
```

Comment:
Maybe include a note that the shared vpc status i managed out-of-band.

Reply:
Addressed at https://github.com/terraform-google-modules/terraform-google-project-factory/pull/91/files#diff-963d4c8c343b85de0fee346ecdad2ff9R63

Reply:
```suggestion
  # The provided project must already be a Shared VPC host
  shared_vpc_host = "false"
```